cli/tests/pcluster/schemas/test_cluster_schema.py

```diff
@@ -1435,3 +1435,53 @@ def test_queue_tag_schema(mocker, config_dict, failure_message):
     else:
         conf = QueueTagSchema().load(config_dict)
         QueueTagSchema().dump(conf)
+
+
+@pytest.mark.parametrize(
+    "config_dict, failure_message, expected_job_exc_inst_alloc",
+    [
+        (
+            {
+                "Name": "Queue",
+                "Networking": {"SubnetIds": ["subnet-12345678"]},
+                "ComputeResources": [{"Name": "compute_resource", "InstanceType": "t2.micro"}],
+                "JobExclusiveInstanceAllocation": True,
```

Comment:
What about  `JobExclusiveInstanceAllocation` = "true", "false", "", 0, 1 and None?

Reply:
Added.
It's worth noting that Marshamallow [fields.Bool](https://marshmallow.readthedocs.io/en/stable/marshmallow.fields.html#marshmallow.fields.Boolean) uses "truthiness" in a very general sense i.e. it evaluates the truthiness of a value, instead of only True/False.